src/components/app/pageBillDetails/filters.tsx

```
@@ -14,10 +14,9 @@ import {
   DTSI_PersonPoliticalAffiliationCategory,
   DTSI_PersonRoleCategory,
 } from '@/data/dtsi/generated'
-import { getValues } from '@/utils/shared/getEntries'
 import { cn } from '@/utils/web/cn'
 
-export const STANCE_OPTIONS = ['All', ...getValues(DTSI_BillPersonRelationshipType)] as const
+export const STANCE_OPTIONS = ['All', ...Object.values(DTSI_BillPersonRelationshipType)] as const
```

why was getValues deleted? sorry, still getting acquainted to the codebase

---

oh, Object.values is generic now, is that it?

---

Per Lucas request https://github.com/Stand-With-Crypto/swc-web/pull/918#discussion_r1629853196.

The util function could suppress errors and the return type is the same as Object.values.